lib/SILOptimizer/SILCombiner/SILCombinerApplyVisitors.cpp

```diff
@@ -791,8 +791,8 @@ struct ConcreteArgumentCopy {
     assert(!paramInfo.isIndirectMutating()
            && "A mutated opened existential value can't be replaced");
 
-    if (!paramInfo.isConsumed())
-      return None;
+//    if (!paramInfo.isConsumed())
```

Comment:
I don't really see a reason for this to be here. After removing it, I didn't get any errors. I'll run the whole test suit to make sure it's OK. @atrick do you see any reason this should remain?

Reply:
That's all good to know. I originally thought this was necessary to prevent regression but after I realized the sil test wasn't checking the specialized function this no longer seems necessary. I've added it back.